octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
```

Comment:
[Severity: Minor] The package declaration is correct, but consider adding a brief comment about the purpose of this class.
Adding a comment at the top of the class can help others (and your future self) understand the purpose of the code quickly.
Rationale: Comments improve code readability and maintainability.
Reference: https://www.oracle.com/java/technologies/javase/codeconventions-contents.html